docs/source/contributing.rst

```diff
@@ -392,7 +392,7 @@ If github shows a red ``X`` next to your latest commit, it means one of our chec
 
 ::
 
-    black --target-version py37 --skip-string-normalization mirdata/
+    black --target-version py36 --target-version py37 --target-version py38 --skip-string-normalization mirdata/ tests/
```

Comment:
For me, the only way I was able to pass the formatting CircleCI tests was by running the black formatting without the `--skip-string-normalization` flag. 
Is this supposed to be an error?

Reply:
I don't know :S

Reply:
Good catch, and sorry for the troubles it caused!

I think it should be simply:
```black --target-version py38 mirdata tests```
meaning that we no longer want to skip the string normalization, and we'll have the formatting target just python 3.8.
(it's how it's running now in `tox.ini` as of #421 )